### ISSUE TITLE
LLM vs transformer naming in doc

### ISSUE THREAD

**Issue by PierreGtch** [braindecode/braindecode#863]
As discussed with Giulia Lioi @glioi at NeurIPS, let's get rid of the LLM term in the models classification.  It is confusing. 

Instead, we label them all transformers and on the pre-trained ones, we add a "foundation model" or "pretrained" label

**Comment by bruAristimunha** [comment]
We don't use the LLM term; instead, we use the large language model term, following the terminology used in the literature. @glioi, can we discuss within the conference?

**Comment by bruAristimunha** [comment]
I am not supporting this without discussing

**Comment by glioi** [comment]
Yes! Let s meet and discuss this!

**Comment by bruAristimunha** [comment]
sorry, i noticed the typo now, I mean Large Brain Model.

https://arxiv.org/abs/2405.18765

**Comment by bruAristimunha** [comment]
The naming convention should be:
What is currently called the 
* "Large Brain Model" will be renamed: the "Foundation model"
* "Small Attention" should be named "Attention/Transformer"

We should also emphasize that it "Foundation model" requires a pre-processing step to function.

The renaming should occur in the documentation.rst file, in the table with the model meta-information, in the model documentation section, and in any plots.

And also in the tags of all models.

**Comment by bruAristimunha** [comment]
Hey @glioi!

I renamed everything as we discussed last week. 

Once the CI work is complete, you will be able to view it on the https://braindecode.org/dev page. When we release the next update, you should be able to see it on the stable page at braindecode.org.

Our release plan is for next week, Friday, when the Brain&AI model PR is merged, and the Reve PR is merged.

I'm creating promotional material to highlight Reve in the release; if the PR #866 is merged by Friday, you should receive it in Google Docs or Canvas at the beginning of the week, or when after I come back to Paris next Tuesday :)

**Comment by glioi** [comment]
Hi @bruAristimunha !
Thanks a lot for the update!

**Comment by glioi** [comment]
concerning REVE, It would be great to add a tutorial on how to use REVE on braindecode. We already have a [colab tutorial ](https://colab.research.google.com/drive/13wMytamrucmU9c2OIc-TcoEIj6xSqPGo?usp=sharing) actually: we just need to adapt it to braindecode!

Let me know what do you need for that!

Have a safe trip back to Paris ;)

Giulia
